.github/dependabot.yml

```diff
@@ -19,6 +19,16 @@ updates:
       timezone: "Europe/Amsterdam"
     reviewers:
       - "infonl/dimpact"
+    ignore:
+      # Ignore updates for these packages because it is needed for our current WildFly version (26.x)
```

Comment:
I would say:

"Ignore updates for these packages because they are provided by the WildFly application server."

Reply:
:spongemock:

Reply:
Changed